### PR TITLE
Run release tests as non-admin user

### PR DESCRIPTION
**Pull request by praveen4g0**
* Helps you to setup imagepuller role at user level (To run release-tests in stage/pre-stage env)
* Update `oc create ns <..>` to `oc new-project <...>`
* organized steps better

**NOTE:**
- we need to create cluster-role and role-bindings to user. 
**CR.yaml**
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: user-cluster-role
rules:
- apiGroups:
  - operator.tekton.dev
  resources:
  - '*'
  verbs:
  - watch
  - get
  - list
- apiGroups:
  - apps
  resources:
  - deployments
  verbs:
  - get
```

**CRB.yaml**
```
apiVersion: rbac.authorization.k8s.io/v1
# This cluster role binding allows anyone in the "manager" group to read secrets in any namespace.
kind: ClusterRoleBinding
metadata:
  name: user-cluster-role-binding
subjects:
- kind: User
  name: <any-user>
  apiGroup: rbac.authorization.k8s.io
roleRef:
  kind: ClusterRole
  name: user-cluster-role
  apiGroup: rbac.authorization.k8s.io
```

### CODE REVIEW

File: pkg/k8s/k8s.go [hunk 1]
```diff
@@ -38,10 +39,9 @@ func NewClientSet() (*clients.Clients, string, func()) {
 	ns := names.SimpleNameGenerator.RestrictLengthWithRandomSuffix("releasetest")
 	cs, err := clients.NewClients(config.Flags.Kubeconfig, config.Flags.Cluster, ns)
 	assert.FailOnError(err)
-	CreateNamespace(cs.KubeClient, ns)
```
Comment (ppitonak) [inline pkg/k8s/k8s.go:41]: Why didn't you remove this function?

File: pkg/k8s/k8s.go [hunk 2]
```diff
@@ -38,10 +39,9 @@ func NewClientSet() (*clients.Clients, string, func()) {
...
-
+	oc.CreateNewProject(ns)
 	return cs, ns, func() {
-		DeleteNamespace(cs.KubeClient, ns)
```
Comment (ppitonak) [inline pkg/k8s/k8s.go:44]: Why didn't you remove this function?

File: pkg/k8s/k8s.go (hunk 1 above)
Comment (praveen4g0) [reply pkg/k8s/k8s.go:41]: ahh,... will do cleanup

File: pkg/k8s/k8s.go [hunk 3]
```diff
@@ -171,22 +171,6 @@ func WaitForDeployment(kc kubernetes.Interface, namespace, name string, replicas
 	return err
 }
 
-func CreateNamespace(kc *clients.KubeClient, ns string) {
```
Comment (praveen4g0) [inline pkg/k8s/k8s.go:174]: Deleted unused functions

**Review by ppitonak — APPROVED** [review]

**Comment by ppitonak** [comment]
It's ready to be merged. Should this PR wait until you create PR with role bindings or can we merge it immediately?

**Comment by praveen4g0** [comment]
no @ppitonak  that pr is independent of it, we shall merge it so atleast , it won't be blocker to `Add Triggers scenarios #16`

**Pull request merged**